src/utils/FontManager.js

```diff
@@ -77,7 +77,11 @@ export async function fixupColorFonts() {
     }
 
     if (await isColrFontSupported()) {
-        const path = `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`;
+        let path = `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`;
+
+        // Normalize slashes because Windows is made of fail
+        path = path.replace(/\\/g, '/');
```

Comment:
should we rather do this in the webpack config script for all requires?

Reply:
is that a thing we have? I wasn't sure ._.